File: BibliotecaPackageKT/libro.kt
package Biblioteca

class Libro(val titulo: String, val autor: String, var ejemplaresDisponibles: Int) {

    fun prestar() {
        if (ejemplaresDisponibles > 0) {
            ejemplaresDisponibles--
            println("Libro prestado: $titulo, Autor: $autor")
        } else {
            println("No hay ejemplares disponibles de $titulo")
        }
    }

    fun devolver() {
        ejemplaresDisponibles++
        println("Libro devuelto: $titulo, Autor: $autor")
    }

    fun informacion() {
        println("Título: $titulo, Autor: $autor, Ejemplares Disponibles: $ejemplaresDisponibles")
    }
}

File: UF2-Programacio_Modular/PyBiblioteca/libro.py
class Libro:
    def __init__(self, titulo: str, autor: str, cantidad: int):
        self.titulo = titulo
        self.autor = autor
        self.cantidad = cantidad

    def prestar(self) -> None:
        if self.cantidad > 0:
            self.cantidad -= 1
            print(f"Libro prestado: {self.titulo}, autor: {self.autor}")
        else:
            print(f"No hay ejemplares de {self.titulo}")

    def devolver(self) -> None:
        self.cantidad += 1
        print(f"Libro devuelto: {self.titulo}, autor: {self.autor}")

    def informacion(self) -> None:
        print(f"titulo: {self.titulo}, autor: {self.autor}, ejemplares disponibles: {self.cantidad}")
